scripts/partner_batch_imports.py

```diff
@@ -173,6 +173,14 @@ def csv_to_ol_json_item(line):
     b = Biblio(data)
     return {'ia_id': b.source_id, 'data': b.json()}
 
+def is_low_quality_book(book_item):
+    """check if a book item is of low quality"""
+    book_title_breakup = book_item.title.split()
+    if "Notebook" in book_title_breakup and book_item.publisher == "Independently Published":
```

Comment:
You should also check for "notebook" in title , solr is case-insensitive, I believe